### PR TITLE
Themes assets/polish

### PR DESCRIPTION
**Pull request by kwonye**
Addresses #3155 by adding correct assets.

Also tweaks UI to get closer to web version.

Previous: 
screenshot_20150915-141344

Current:
screenshot_20150915-141212

Web:
screen shot 2015-09-15 at 1 19 29 pm

Still not 100% (font issues), but it's closer and adds assets.

Needs review: @tonyr59h

### CODE REVIEW

File: WordPress/src/main/res/layout/theme_grid_cardview_header.xml [hunk 1]
```diff
@@ -28,12 +28,16 @@
                 <org.wordpress.android.widgets.WPTextView
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:textStyle="bold"
+                    android:textColor="@color/grey"
+                    android:textSize="12sp"
```
Comment (tonyr59h) [inline WordPress/src/main/res/layout/theme_grid_cardview_header.xml]: Should use `@dimen/text_sz_small`

**Comment by tonyr59h** [comment]
Very nice changes, it's looking a lot cleaner/closer to the web version :smiley:

The selectable background looks nice on the list items. There's a small issue of the press animation not extending below the item view, looks like there's [no bottom margin defined](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/3155-themes-assets/WordPress/src/main/res/layout/theme_grid_item.xml#L13) in the layout. Can you use that same background for the Customize/Details/Support buttons?

**Comment by kwonye** [comment]
> The selectable background looks nice on the list items. There's a small issue of the press animation not extending below the item view, looks like there's no bottom margin defined in the layout. Can you use that same background for the Customize/Details/Support buttons?

Sorry about the confusion. All selection 'fixes' is on the other branch.

**Comment by tonyr59h** [comment]
:+1: Gonna :shipit: after Travis has his way.

**Pull request merged**